cypress/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+
+module.exports = {
+  extends: ['../.eslintrc.js', 'plugin:cypress/recommended', 'prettier'],
+  parserOptions: {
+    sourceType: 'module',
+  },
+  plugins: ['cypress', 'no-only-tests'],
```

Comment:
The [no-only-tests](https://www.npmjs.com/package/eslint-plugin-no-only-tests) plugin is to prevent accidentally committing tests that disable the whole test suite by preventing the use of e.g. `it.only('some test', () => {})`.